<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="Seachange-PC" name="Jibe_PURC_ADMIN.Function_DepartmentTest" tests="8" failures="1" timestamp="11 Sep 2017 03:58:26 GMT" time="157.416" errors="0">
  <testcase name="FunctionDepartmentURL" time="7.687" classname="Jibe_PURC_ADMIN.Function_DepartmentTest"/>
  <testcase name="ValidationforAddFunctionDepartment" time="41.297" classname="Jibe_PURC_ADMIN.Function_DepartmentTest"/>
  <testcase name="DuplicateAddFunctionDepartment" time="19.281" classname="Jibe_PURC_ADMIN.Function_DepartmentTest"/>
  <testcase name="SearchAddFunctionDepartment" time="12.848" classname="Jibe_PURC_ADMIN.Function_DepartmentTest"/>
  <testcase name="EditFunctionDepartment" time="28.441" classname="Jibe_PURC_ADMIN.Function_DepartmentTest"/>
  <testcase name="SearchEditedFunctionDepartment" time="12.769" classname="Jibe_PURC_ADMIN.Function_DepartmentTest"/>
  <testcase name="DeletetFunctionDepartment" time="15.941" classname="Jibe_PURC_ADMIN.Function_DepartmentTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertTrue(Assert.java:41)
	at org.testng.Assert.assertTrue(Assert.java:51)
	at Jibe_PURC_ADMIN.Function_DepartmentTest.DeletetFunctionDepartment(Function_DepartmentTest.java:166)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </failure>
  </testcase> <!-- DeletetFunctionDepartment -->
  <testcase name="SearchDeletetFunctionDepartment" time="19.152" classname="Jibe_PURC_ADMIN.Function_DepartmentTest"/>
</testsuite> <!-- Jibe_PURC_ADMIN.Function_DepartmentTest -->
